modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/upgrade/registry/SXPServiceUpgradeStepRegistrator.java

```diff
@@ -51,21 +51,46 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 		registry.register(
 			"1.3.0", "1.3.1",
 			new com.liferay.search.experiences.internal.upgrade.v1_3_1.
-				SXPBlueprintUpgradeProcess());
+				DummyUpgradeProcess());
 
 		registry.register(
 			"1.3.1", "1.3.2",
 			new com.liferay.search.experiences.internal.upgrade.v1_3_2.
-				SXPBlueprintUpgradeProcess());
+				DummyUpgradeProcess());
 
 		registry.register(
 			"1.3.2", "1.3.3",
 			new com.liferay.search.experiences.internal.upgrade.v1_3_3.
-				SXPBlueprintUpgradeProcess());
+				DummyUpgradeProcess());
 
 		registry.register(
 			"1.3.3", "2.0.0",
 			new com.liferay.search.experiences.internal.upgrade.v2_0_0.
+				DummyUpgradeProcess());
+
+		registry.register(
+			"1.3.0", "2.0.0",
+			new com.liferay.search.experiences.internal.upgrade.v2_0_0.
+				DummyUpgradeProcess());
```

Comment:
@gustavolimav Maybe this is not necessary

Reply:
Pushed already